deepcell/utils/plot_utils.py

```diff
@@ -59,16 +59,19 @@ def get_js_video(images, batch=0, channel=0, cmap='jet', vmin=0, vmax=30):
         'cmap': cmap,
     }
 
+    if vmax == 0:
+        vmax = images.max()
+
     # TODO: do these not work for other cmaps?
-    if cmap == 'cubehelix':
+    if cmap == 'cubehelix' or cmap == 'jet':
```

Comment:
Do you know what's up with the color maps? Should these be enumerated, or just check if `cmap is None`?

Reply:
I'm not sure what the TODO is referencing. Maybe there are certain cmaps it doesn't work with and it was safer to do this than try and list all the exceptions? Not really sure, and haven't needed any other cmaps so far